include/aws_iot_mqtt_client.h

```diff
@@ -61,6 +61,7 @@ extern "C" {
 #include "threads_interface.h"
 #endif
 
+/** Greatest packet identifier, per MQTT spec */
```

Comment:
Do we need periods here?

Reply:
The documentation for this version does not put periods at the end of lines.  So for consistency, no.

Reply:
https://github.com/aws/aws-iot-device-sdk-embedded-C/blob/v4_beta/libraries/standard/mqtt/include/types/iot_mqtt_types.h#L1129

Reply:
That's v4 though.  v3 does not do that.